jcommune-service/src/main/java/org/jtalks/jcommune/service/nontransactional/BBCodeService.java

```diff
@@ -73,13 +74,21 @@ public String convertBbToHtml(String bbEncodedText) {
         for (TextProcessor preprocessor : preprocessors) {
             bbEncodedText = preprocessor.process(bbEncodedText);
         }
-        bbEncodedText = processor.process(bbEncodedText);
+        if (isUserbbCode(bbEncodedText)){
```

Comment:
Could you please add comments with the details? In general all non-typical solutions should have solid comments.